scripts/prepdocslib/blobmanager.py

```diff
@@ -63,6 +63,15 @@ async def upload_pdf_blob_images(
         start_time = datetime.datetime.now(datetime.timezone.utc)
         expiry_time = start_time + datetime.timedelta(days=1)
 
+        font = None
+        try:
+            font = ImageFont.truetype("arial.ttf", 20)
+        except OSError:
+            try:
+                font = ImageFont.truetype("/usr/share/fonts/truetype/freefont/FreeMono.ttf", 20)
```

Comment:
We are hardcoding a Unix style path which is guaranteed to fail if this is running on Windows

Reply:
That first one, "arial.ttf", is supposed to work on Windows, so in theory most Windows machines wouldn't go down that path? I agree that the hardcoded paths doesnt seem great for Unix, and there's probably a more elegant situation overall.
I don't know if theres a TTF file we're allowed to ship with the repo, I mentioned to Shakir but he wasn't sure.